spec/models/commit_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Commit, type: :model do
```

Comment:
It should behave like a `'an ActiveModel compatible object'` (see, for example, the repository_spec.rb)